__init__.py

```diff
@@ -1,2 +1,2 @@
 # This file is only necessary so that it can be used as a git submodule in https://github.com/getpelican/pelican-plugins
-from .linkclass import *
+from .pelican_linkclass import *
```

Comment:
Could make this an absolute import instead of a relative one please ?
This should not make any difference except make this plugin tests pass when included in `getpelican/pelicans-plugins` with `pytest`.
_cf._ https://github.com/rlaboiss/pelican-linkclass/pull/7